pandas/core/frame.py

```diff
@@ -2295,6 +2295,138 @@ def _from_arrays(
         )
         return cls(mgr)
 
+    @doc(storage_options=generic._shared_docs["storage_options"])
+    def to_rdata(
+        self,
+        path_or_buffer: FilePathOrBuffer,
+        file_format: str = "infer",
+        rda_name: str = "pandas_dataframe",
```

Comment:
Something closer to what `read_excel` does where ti chooses sheets[0] by default probably makes more sense.

Reply:
Not clear what you mean here. Currently, librdata supports only writing a single `DataFrame`  to either R data file format. Also, `read_rdata` (not shown here in `frame.py`) maps R object names to dict keys.

Reply:
All readers (AFAICT) pull in a single dataframe, including read_excel even if the sheet has multiple copies.  Perhaps the default should be to return the first DataFrame-like object in the rdata file.  This is more-or-less what ExcelReader does.